Assignment/Program.cs

```diff
@@ -1,7 +1,49 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿for (; ; ) {
+   Console.Write ("Enter a number : ");
+   string decimalNumber = Console.ReadLine ();
+   if (int.TryParse (decimalNumber, out int decimalNum)) {
+      //Decimal to Binary

```

Comment:
Always leave a space after the slashes in comment
// Decimal to Binary

Reply:
ok